runtimes/runtimes.go

```diff
@@ -480,6 +480,21 @@ var RUNTIME_DETAILS = []byte(`{
                     "attachmentName": "codefile",
                     "attachmentType": "text/plain"
                 }
+            },	    
+            {
+                "kind": "dotnet:3.1",
```

Comment:
Please add unit tests and perhaps an integration test to validate support in our CI/CD process. Thanks!